### ISSUE TITLE
S5 KLTE 900T Kernel?

### ISSUE THREAD

**Issue by tomford007** [offensive-security/nethunter-devices#140]
Please fill out this issue report and attach a copy of dmesg (`adb exec-out dmesg > dmesg.log`)  
You should run the dmesg command immediately after experiencing your problem!  
You can post your dmesg.log on a pastebin site or upload it to Google drive.
  
For installer issues, consider opening an issue report at https://github.com/offensive-security/kali-nethunter instead.  

## Issue:

Is there a kernel download for my device? Thanks
CM14 on Nougat 7.1

**Comment by xroot25** [comment]
Hey, I saw your post about the nethunter kernel. If you are attempting to
use monitor mode on a klte, I have noticed that it will not work, if you're
using anything past marshmallow. I have been successful in promiscuous mode
only with a special ROM that you can find on XDA, of mm, called the temasec
room mm 6.0.1, and then the kernel from the site. Hope it helps, I have the
exact same model.

**Comment by tomford007** [comment]
Hi, thank you for responding. I would apply this rom on top of nougat? I can't get my S5 to recognize my external wifi adapters at all..

**Comment by xroot25** [comment]
No. You will never succeed in NOUGAT and nethunter. It is due to either the
unofficial nougat for klte or changes between marshmallow and nougat. You
need to clean wipe the device via twrp, and install the temasec version of
CyanogenMod for klte, and then reboot twrp, and install the kernel and
nethunter files via zip. I have spent many months in Discovery of this
tidbit I relay to you, and it's the only solution to wireless adapter
support. It is well worth it, as it is spectacular in it's actuality.
Better than anything else for hacking.  If you have any trouble, I'd be
willing to locate the room links.

On Mar 26, 2018 11:29 PM, "tomford007" <notifications@github.com> wrote:

Hi, thank you for responding. I would apply this rom on top of nougat? I
can't get my S5 to recognize my external wifi adapters at all..

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/offensive-security/nethunter-devices/issues/140#issuecomment-376412199>,
or mute the thread
